tests/test_language.py

```diff
@@ -64,4 +64,4 @@ def test_name_for_languageset(self):
         eq_(u"English/español", m(["eng", "spa"]))
         eq_(u"español/English", m("spa,eng"))
         eq_(u"español/English/Chinese", m(["spa","eng","chi"]))
-        assert_raises(ValueError(m, ["eng, nxx"]))
+        assert_raises(ValueError, m, ["eng, nxx"])
```

Comment:
This test wasn't doing anything  because of a misplaced closing parenthesis-- it was basically `assert_raises(ValueError())`. That's not allowed in the Python 3 version of nose -- you have to actually run some code or the assertion will fail.